cppcache/integration-test/testThinClientPdxInstance.cpp

```diff
@@ -1260,8 +1259,7 @@ DUNIT_TASK_DEFINITION(CLIENT2, modifyPdxInstance)
     ASSERT(newPiPtr->hasField("m_int32") == true, "m_int32 = true expected");
     newVal = newPiPtr->getIntField("m_int32");
     ASSERT(val + 1 == newVal, "val + 1 == newVal expected");
-    ASSERT((*pIPtr.get() == *newPiPtr.get()) == false,
-           "PdxInstance should not be equal");
+    ASSERT((*pIPtr == *newPiPtr) == false, "PdxInstance should not be equal");
```

Comment:
The test code seems very fond of verbose comparisons like this - how about just `(*pIPtr != *newPiPtr)`?

Reply:
Again, this was mostly automated. I'd like to keep it that way.